EosioSwiftTests/RpcTestConstants.swift

```diff
@@ -1883,6 +1885,172 @@ public class RpcTestConstants {
     ]
     """
 
+    public static let sendTransActionResponseJson = """
+    {
+        "processed": {
+            "account_ram_delta": null,
+            "action_traces": [
+                {
+                    "account_ram_deltas": [],
+                    "act": {
+                        "account": "eosio.token",
+                        "authorization": [
+                            {
+                                "actor": "swdev1212121",
+                                "permission": "active"
+                            }
+                        ],
+                        "data": {
+                            "from": "swdev1212121",
+                            "memo": "",
+                            "quantity": "1.0000 EOS",
+                            "to": "paulk1111111"
+                        },
+                        "hex_data": "1044104184ad12c7104208210418b5a9102700000000000004454f530000000000",
+                        "name": "transfer"
+                    },
+                    "action_ordinal": 1,
+                    "block_num": 56110902,
+                    "block_time": "2019-10-22T15:32:46.000",
+                    "closest_unnotified_ancestor_action_ordinal": 0,
+                    "console": "",
+                    "context_free": false,
+                    "creator_action_ordinal": 0,
+                    "elapsed": 65,
+                    "error_code": null,
+                    "except": null,
+                    "producer_block_id": null,
+                    "receipt": {
+                        "abi_sequence": 4,
+                        "act_digest": "fc7afe8d172f698189449bf95d6a2a193e27759a264e21741e00872a48a5ec39",
+                        "auth_sequence": [
+                            [
+                                "swdev1212121",
+                                28
+                            ]
+                        ],
+                        "code_sequence": 5,
+                        "global_sequence": 498392582,
+                        "receiver": "eosio.token",
+                        "recv_sequence": 76627315
+                    },
+                    "receiver": "eosio.token",
+                    "trx_id": "2e611730d904777d5da89e844cac4936da0ff844ad8e3c7eccd5da912423c9e9"
+                },
+                {
+                    "account_ram_deltas": [],
+                    "act": {
+                        "account": "eosio.token",
+                        "authorization": [
+                            {
+                                "actor": "swdev1212121",
+                                "permission": "active"
+                            }
+                        ],
+                        "data": {
+                            "from": "swdev1212121",
+                            "memo": "",
+                            "quantity": "1.0000 EOS",
+                            "to": "paulk1111111"
```

Comment:
so every time unit tests are run, I'll get 1 EOS? nice! :)

what happens if the test account runs out of tokens?

Reply:
It's a mocked RPC Provider so unfortunately I don't think you are getting rich off of the unit tests.